styles/style.css

```diff
@@ -0,0 +1,571 @@
+
+.nav__home {
+  display: flex;
+  align-items: center;
+}
+.nav__logo_mob { width: 6.5rem; }
+
+.nav__logo_desk {
+  width: calc(6.45rem - 4.13vw);
+  min-width: 4.58em;
+}
+
+.nav__hamb {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-self: flex-end;
+}
+
+.nav__bar {
+  width: 2.56rem;
+  height: 0.3125rem;
+}
+
+.nav__bar_black { background: black; }
+
+.nav__amount {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 1rem;
+  width: 1rem;
+  color: #fff;
+  background: #000;
+  border-radius: 50%;
+  margin: .3em;
+}
+
+.nav__checkbox_none,
+.nav__items_none { display: none; }
+
+.nav__checkbox:checked ~ .nav__items { display: grid; }
+
+.nav__home_desk { display: none; }
+
+.header {
+  height: 11.94rem;
+  display: grid;
+  grid-template-rows: 5.4em 6.3em;
+
+  /* background: url(../logos/rob-potter-422534-unsplash.png) no-repeat -7.5em/cover; */
+  background: url(../logos/rob-potter-422534-unsplash.png) no-repeat calc(50.18vw - 20.88em)/cover;
+  color: #fff;
+}
+
+.header__title { grid-area: 2; }
+
+.header__link {
+  display: flex;
+  grid-area: 3;
+  justify-self: start;
+  align-self: flex-start;
+}
+
+.info__link { display: none; }
+
+.product {
+  display: grid;
+  height: 29.9em;
+  grid-template-rows: 2.8em min-content;
+}
+
+.product__header {
+  grid-area: 2;
+  text-align: center;
+  padding-right: .5em;
+}
+
+.product__items {
+  grid-area: 3;
+  display: flex;
+  justify-content: center;
+}
+
+.product__item {
+  display: grid;
+  height: 18.3em;
+  width: 22.24em;
+  margin-left: 1.2em;
+  position: relative;
+}
+
+.product__item:nth-child(2),
+.product__item:nth-child(3),
+.product__item:nth-child(4) { display: none; }
+
+.product__link {
+  display: grid;
+  grid-template-rows: 14.9em 1.6em;
+}
+
+.product__clrs {
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  width: 6.2em;
+  justify-self: end;
+  align-self: end;
+  align-items: center;
+}
+
+.product__img {
+  width: 13.55em;
+  height: 12.8em;
+  justify-self: center;
+}
+
+.product__clr {
+  width: 1.25em;
+  height: 1.25em;
+  border-radius: 50%;
+}
+
+.product__clr_red { background: #b62e3c; }
+
+.product__clr_bl { background: #0e1211; }
+
+.product__clr_gr { background: #c4c4c4; }
+
+.product__clr_yl { background: #fdb431; }
+
+.product__clr_cm { background: #e29d69; }
+
+.product__clr_br { background: #762e31; }
+
+.product__clr_blu { background: #41479b; }
+
+.product__clr_lb { background: #257da9; }
+
+/* .footer {height: 56.125rem;} */
+
+@media only screen and (min-width: 375px) {
+  .header__title { font-size: calc(7.63vw - .719em); }
+
+  .header__link,
+  .story__link { font-size: calc(.364em + 2.035vw); }
+
+  .product__header,
+  .info__header,
+  .stories__header { font-size: calc(.87em + 2.54vw); }
+
+  .product__name { font-size: calc(.85em + 1.27vw); }
+
+  .product__price,
+  .story__header { font-size: calc(1.27em + 1.27vw); }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/r4/pullRequest?prid=5887702)